microservices/embeddingservice/src/embed_text.py

```diff
@@ -43,37 +44,46 @@ def download_model(
 class EmbedText:
     def __init__(self, config):
         self.config = config
+        self.model_config = config["models"]["c2c_models"]
         self.models = {}
         bucket = None
         if sa := self.config.get("service_account"):
             credentials = service_account.Credentials.from_service_account_info(sa)
             client = storage.Client(credentials=credentials)
             bucket = client.bucket(self.config["bucket_name"])
-        for model in self.config["models"]:
-            for model_name, model_path in model.items():
-                bucket_path = (
-                    f'{self.config["bucket_path"]}/{model_path.split("/")[-1]}.zip'
-                )
+        for model in self.model_config:
```

Comment:
iterating over `self.model_config.items()` is probably more concise

Reply:
Done